pkg/universe.dagger.io/python/test/test.cue

```diff
@@ -3,13 +3,28 @@ package python
 import (
 	"dagger.io/dagger"
 	"dagger.io/dagger/core"
-
+	"universe.dagger.io/docker"
 	"universe.dagger.io/python"
 )
 
 dagger.#Plan & {
 	actions: test: {
 
+		// Run with a custom path to python
+		customPath: {
+			
+			// python:3.10-alpine has both python3 and python on the PATH
+			_image: docker.#Pull & {
+				source: "python:3.10-alpine"
+			}
+			run: python.#Run & {
+				input: _image.output
+				command: name: "python"
+				script: contents: #"print("Hello, world!")"#
+			}
+			// This needs no output test because it is only testing that the command runs
```

Comment:
I agree with your comment but I think it would be more accurate to verify the output so we are sure that the command run and everything went good.

Reply:
I actually disagree. The underlying `docker.#Run` will stop execution if the container exited non-zero and that's all we care about. If the user (or the test, in this case) provides a path that does not properly evaluate a Python file but also does not error, this test exists to make sure that the action still _runs_. Checking the output of that action is irrelevant.